src/components/feedback-button.js

```diff
@@ -0,0 +1,38 @@
+import { css } from 'react-emotion'
+import { faCommentDots } from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import React from 'react'
+import { BLUE, LIGHT_BLUE, WHITE } from '../assets/colors'
+
+export default function FeedbackButton () {
```

Comment:
do you dislike arrow functions? it's pretty much the norm now with ES6

Reply:
not a blocking comment

Reply:
I don't dislike them. They are about the same, with arrow functions changing the scope of the function.